src/libsyncengine/jobs/network/getappversionjob.cpp

```diff
@@ -69,6 +70,12 @@ std::string GetAppVersionJob::getContentType(bool &canceled) {
     return {};
 }
 
+void GetAppVersionJob::setQueryParameters(Poco::URI &uri, bool &canceled) {
+    for (const auto id: _userIdList) {
```

Comment:
```suggestion
    for (const auto &id: _userIdList) {
```

Reply:
Done in c6700de4a937df1ac5a2c4300b373e5c7e1a31c1